docs/installation/introduction.rst

```diff
@@ -1,6 +1,125 @@
+emphasizes ease of installation, while providing a high stability for the
+scheduled workloads:
+
+- One machine running Bootstrap and control plane services
+- Two other machines running control plane and Infra services
+- Three more machines for workload applications
+
+.. image:: img/standard-arch.png
+   :width: 100%
+
+Machines dedicated to the control plane do not need large
+amounts of resources (see
+:ref:`the sizing notes below<installation-intro-sizing>`, and can safely run as
+virtual machines. Running workloads on dedicated machines also allows for
+simpler sizing of said machines, as MetalK8s impact would be negligible.
+
+Extended Architecture
+"""""""""""""""""""""
+MetalK8s is designed with on-premise, offline, reliable deployments in mind.
+This example architecture focuses on reliability rather than compacity,
+offering the finest control over the entire platform:
+
+- One machine dedicated to running Bootstrap services (see
+  :ref:`the Bootstrap role<node-role-bootstrap>` definition below)
+- Three extra machines (or five if installing a really large cluster,
+  100+ nodes) for running the Kubernetes_ control plane (with
+  :ref:`core K8s services<node-role-master>` and the backing
+  :ref:`etcd DB<node-role-etcd>`)
+- One or more machines dedicated to running Infra services (see
+  :ref:`the Infra role<node-role-infra>`)
+- Any number of machines dedicated to running applications, the number and
+  :ref:`sizing<installation-intro-sizing>` depending on the applications (for
+  instance, Zenko_ would recommend using three or more machines)
+
+.. image:: img/extended-arch.png
+   :width: 100%
+
+.. _Zenko: https://zenko.io/
+
+.. _installation-intro-compact-arch:
+
+Compact Architectures
+"""""""""""""""""""""
+While not being focused on having the smallest compute and memory footprints,
+MetalK8s can provide a fully functional single node "cluster". The Bootstrap
+node can be configured to also allow running applications next to all the other
+services required (see :ref:`the section about taints<node-taints>` below).
+
+A single node cluster does not provide any form of resilience to machine or
+site failure, which is why the recommended most compact architecture to use in
+production includes three machines:
+
+- Two machines running control plane services alongside infra and workload
+  applications
+- One machine running Bootstrap services in addition to all the other services
+
+.. image:: img/compact-arch.png
+   :width: 100%
+
+Please note that sizing of such compact clusters needs to account for the
+expected load, and the exact impact of colocating an application with MetalK8s
+services needs to be evaluated by said application's provider.
+
+Variations
+""""""""""
+It is possible to customize the chosen architecture using combinations of
+:ref:`roles<node-roles>` and :ref:`taints<node-taints>`, which are described
+below, to adapt to the available infrastructure.
```

Comment:
Customize the chosen architecture to adapt the available infrastructure using combinations of :ref:`roles<node-roles>` and :ref:`taints<node-taints>`.
Described below == reference/link? Does it refers to node taints and roles or the two paragraphs below?

Reply:
The references are set on `roles` and `taints` already. Not clear enough?